### PR TITLE
Updated the SBOM Drift

### PR DESCRIPTION
**Pull request by lavakush07**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Comment by rohanmaharjan100** [comment]
@lavakush07 
image

**Comment by lavakush07** [comment]
> @lavakush07 image

But it is working fine on local. The paths are correct.

**Comment by rohanmaharjan100** [comment]
@lavakush07  I think you forgot to add images to the static folder. Please make sure static folder has the required images.

File: docs/software-supply-chain-assurance/sbom/sbom-drift.md [hunk 1]
```diff
@@ -39,13 +39,12 @@ To set any artifact version as baseline, go to artifact view and click on three
 
 Click on “Initiate Drift Analysis” in the three dots menu on the artifact version  
 
-<docimage path={require('../static/Initiate-Drift-Analysis.png')} />
-
+<DocImage path={require('../static/initiate-drift-analysis.png')} width="90%" height="60%" title="Click to view full size image" />
```
Comment (rohanmaharjan100) [inline docs/software-supply-chain-assurance/sbom/sbom-drift.md:42]: The case for the image name appears incorrect. It should be: `../static/Initiate-Drift-Analysis.png`.

**Review by rohanmaharjan100 — COMMENTED** [review]
case for the image name looks wrong. It should be =>docs/software-supply-chain-assurance/static/Initiate-Drift-Analysis.png

**Comment by lavakush07** [comment]
> case for the image name looks wrong. It should be =>../static/Initiate-Drift-Analysis.png

In my local it is the same. I pushed the changes with the correct path. 
Screenshot 2024-12-23 at 9 01 20 PM

File: docs/software-supply-chain-assurance/sbom/sbom-drift.md (hunk 1 above)
Comment (wei-harness) [reply docs/software-supply-chain-assurance/sbom/sbom-drift.md:42]: suggest to use kebab case for static files, e.g.: `initiate-drift-analysis.png`, please rename the png file instead.
@rohanmaharjan100 please add a GHA rule for it.

**Review by wei-harness — CHANGES_REQUESTED** [review]

**Review by tejakummarikuntla — APPROVED** [review]

**Review by wei-harness — APPROVED** [review]

**Pull request merged**